core-primitives/types/src/lib.rs

```diff
@@ -58,16 +67,6 @@ pub struct Request {
 	pub cyphertext: Vec<u8>,
 }
 
-pub type IpfsHash = [u8; 46];
```

Comment:
Seemed strange to me that some types were defined below struct implementations, so I moved them up.